File: app/src/main/java/com/czy/wififiletransfer/MainActivity.java
package com.czy.wififiletransfer;

import android.os.Bundle;
import android.view.View;

/**
 * 作者：chenZY
 * 时间：2018/4/3 14:56
 * 描述：https://www.jianshu.com/u/9df45b87cfdf
 * https://github.com/leavesC
 */
public class MainActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void startFileSenderActivity(View view) {
        startActivity(FileSenderActivity.class);
    }

    public void startFileReceiverActivity(View view) {
        startActivity(FileReceiverActivity.class);
    }

}

File: app/src/main/java/github/leavesczy/wififiletransfer/MainActivity.kt
package github.leavesczy.wififiletransfer

import android.os.Bundle
import android.view.View
import github.leavesczy.wififiletransfer.receiver.FileReceiverActivity
import github.leavesczy.wififiletransfer.sender.FileSenderActivity

/**
 * @Author: CZY
 * @Date: 2022/9/26 17:10
 * @Desc:
 */
class MainActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        findViewById<View>(R.id.btnReceiver).setOnClickListener {
            startActivity(FileReceiverActivity::class.java)
        }
        findViewById<View>(R.id.btnSender).setOnClickListener {
            startActivity(FileSenderActivity::class.java)
        }
    }

}